docker/database/sql/000_ht_schema.sql

```diff
@@ -0,0 +1,116 @@
+CREATE DATABASE IF NOT EXISTS ht;
```

Comment:
Ultimately I think it will make sense to have a common mysql database docker image for HT apps with these tables pre-populated, but again, I'm fine with duplicating this here for the time being.

Reply:
I would be delighted to see that happen. CRMS Travis builds take forever.